crates/compiler/test_syntax/tests/snapshots/pass/ann_open_union.expr.result-ast

```diff
@@ -15,32 +15,6 @@ Defs(
         spaces: [],
         type_defs: [],
         value_defs: [
-            Annotation(
-                @0-3 Identifier {
-                    ident: "foo",
-                },
-                @6-28 TagUnion {
-                    ext: Some(
-                        @27-28 Wildcard,
-                    ),
-                    tags: [
-                        @7-11 Apply {
-                            name: @7-11 "True",
-                            args: [],
-                        },
-                        @13-26 Apply {
-                            name: @13-20 "Perhaps",
-                            args: [
-                                @21-26 Apply(
-                                    "",
-                                    "Thing",
-                                    [],
-                                ),
-                            ],
-                        },
-                    ],
-                },
-            ),
```

Comment:
I'm a little worried that getting rid of `Annotation` here may break things but all tests passed so :shrug:

Reply:
it's an expected remove caused by this fix:

https://github.com/roc-lang/roc/pull/6888/files#diff-144786552ec43e11867281790c5f2a807ebe7cd50c0323edaa4c0be1488d9f9aR1296-R1308

previously it didn't replace anything but created a new item